packages/space-opera/src/components/file_modal/file_modal.ts

```diff
@@ -36,20 +36,20 @@ export class FileModalElement extends LitElement {
   @property({type: String}) accept = '';
   @query('input#file-input') fileInput!: HTMLInputElement;
 
-  private blobsResolver?: BlobArrayResolver;
+  private filesResolver?: FileArrayResolver;
 
-  open(): Promise<Blob[]|undefined> {
+  open(): Promise<File[]|undefined> {
```

Comment:
Just for my own information, what problem is this solving? You mentioned something about a `.name`; is that causing an error somewhere?

Reply:
When I removed `any`,
https://github.com/google/model-viewer/blob/cbdb5076c55ed55c8cde8e8b656356f1e8204293/packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts#L219
following error occured.
https://github.com/google/model-viewer/blob/cbdb5076c55ed55c8cde8e8b656356f1e8204293/packages/space-opera/src/components/model_viewer_snippet/components/open_button.ts#L225
`Property 'name' does not exist on type 'Blob'.ts(2339)`
<img width="981" alt="スクリーンショット 2021-03-10 8 43 56" src="https://user-images.githubusercontent.com/50798936/110553840-03d33300-817d-11eb-9752-abc953d4c9c5.png">
This is because `this.fileModal.open()` returns `Promise<Blob[]>`.
So, `this.fileModal.open()` should return `Promise<File[]>`

Reply:
Got it, thanks for the clarification!